src/var/flat.js

```diff
@@ -0,0 +1,14 @@
+define( [
+	"./arr",
+	"./concat"
+], function( arr, concat ) {
+	"use strict";
+
+	return arr.flat ? function() {
```

Comment:
Please add a support comment, i.e. a comment starting with `Support:`, mentioning affected browsers & explaining where the fallback is used. Here's an example of a support comment:
https://github.com/jquery/jquery/blob/b334ce7735ae453bd5643b251f36c3cedce4b3de/src/css.js#L323-L328
Here, judging by MDN data:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat
you need to mention IE, Edge & Safari (all with version ranges).

Reply:
Done, please re-check it and tell me if it should be documented some way different.